app/src/main/java/com/appunite/loudius/network/datasource/UserDataSource.kt

```diff
@@ -18,8 +18,6 @@ package com.appunite.loudius.network.datasource
 
 import com.appunite.loudius.network.model.User
 import com.appunite.loudius.network.services.UserService
-import javax.inject.Inject
-import javax.inject.Singleton
```

Comment:
Question: Is it really building in the correct way? We have annotations below and here are deleted imports ;)

Reply:
should be fixed now :)